src/main/apiClient.ts

```diff
@@ -95,7 +95,7 @@ type SingleFactorPasswordlessParams = {
 }
 
 type StepUpPasswordlessParams = {
-  authType: 'magic_link' | 'sms'
+  authType?: 'email' | 'sms'
```

Comment:
This is a breaking change, we should add `email` instead and keep `magic_link` for the time being.

Reply:
How is that breaking since the feature was not available?